index.bs

```diff
@@ -340,6 +340,11 @@ When the <dfn method for="XR">requestSession(|mode|)</dfn> method is invoked, th
             1. If |immersive| is <code>true</code>, set [=pending immersive session=] to <code>false</code>.
             1. Abort these steps.
         1. Let |session| be a new {{XRSession}} object.
```

Comment:
No, because the new line will actually populate an internal variable of the session with the enabled features.

Reply:
Oh I see.  [=Resolve the requested features=] actually changes the session, yes?  If so, then line 348 needs to go above so that the algorithm can reference those properties ;)